Migrated guidance from the DSM to the docs site for M.gov (#2047)
CMSgov/design-system: packages/docs/content/components/tooltip.mdx

  sketchLink: My311v3
---

<ThemeContent onlyThemes={['medicare']}>

> ### Guidance

Reviewer: Let's move this to the guidance section at the bottom of the page

## Guidance

<ThemeContent onlyThemes={['medicare']}>

- Use a dashed underline link design.

> - Be a hover over interaction on desktop and click to activate on mobile.

Reviewer: ```suggestion
- Use a hover over interaction on desktop and click to activate on mobile.
```

> - Appear right next to the linked term.

Reviewer: ```suggestion
- Appear next to the linked term.
```

- Words inside a tooltip won't break over lines for legibility.
- A tooltip link can break over lines, if needed.
- When the tooltip defines a term: Repeat the linked term at the top of the tool tip (bold styling).<br />
 Have space for next line of text be a soft-return (single line spacing).

> - When the tool tip is just contextual help: Don't include the linked term at the top of the tool tip.

Reviewer: ```suggestion
- When the tooltip is just contextual help: Don't include the linked term at the top of the tool tip.
```

> - When the tool tip is just contextual help: Don't include the linked term at the top of the tool tip.

Reviewer: ```suggestion
- When the tooltip is just contextual help: Don't include the linked term at the top of the tooltip.
```

### Content guidelines

- Limit text to no more than 240 characters for optimum user experience.

> - If required to include a legal disclaimer to point to Medicare glossary, the disclaimer isn't included in character count.

Reviewer: ```suggestion
- If required to include a legal disclaimer to point to [Medicare glossary](https://www.medicare.gov/glossary), the disclaimer isn't included in character count.
```
